test/mix/tasks/phx.gen.auth_test.exs

```diff
@@ -84,7 +84,7 @@ defmodule Mix.Tasks.Phx.Gen.AuthTest do
       )
 
       assert_file("config/test.exs", fn file ->
-        assert file =~ "config :bcrypt_elixir, :log_rounds, 1"
+        assert file =~ "config :argon2_elixir, :log_rounds, 1"
```

Comment:
Based on my experience and [`argon2_elixir` docs](https://github.com/riverrun/argon2_elixir/blob/master/lib/argon2/stats.ex#L29), this should be:

```elixir
config :argon2_elixir, t_cost: 1, m_cost: 8
```

Reply:
I'm guessing this would also need to be changed in `test/mix/tasks/phx_gen_auth/integration_tests/default_app_test.exs`